src/cbmc/cbmc_parse_options.cpp

```diff
@@ -85,6 +85,25 @@ ::cbmc_parse_optionst::cbmc_parse_optionst(
 {
 }
 
+void cbmc_parse_optionst::set_default_options(optionst &options)
+{
+  // Default true
+  options.set_option("assertions", true);
+  options.set_option("assumptions", true);
+  options.set_option("built-in-assertions", true);
+  options.set_option("pretty-names", true);
+  options.set_option("propagation", true);
+  options.set_option("sat-preprocessor", true);
+  options.set_option("simplify", true);
+  options.set_option("simplify-if", true);
+
+  // Default false
+  options.set_option("stop-on-fail", false);
```

Comment:
Could you please add `unwinding-assertions`, `partial-loops`, `slice-formula` in here and refactor their current command-line/options code to use `if(cmdline.isset...)`?